pkg/cluster/spec/tiflash.go

```diff
@@ -63,6 +63,7 @@ type TiFlashSpec struct {
 	ResourceControl      meta.ResourceControl `yaml:"resource_control,omitempty" validate:"resource_control:editable"`
 	Arch                 string               `yaml:"arch,omitempty"`
 	OS                   string               `yaml:"os,omitempty"`
+	CPUFlags             string               `yaml:"cpu_flags,omitempty"`
```

Comment:
I don't understand what this field is used for

Reply:
It's used in func `func checkTiFlashCPUFlags(topo spec.Topology, version string) error` https://github.com/solotzg/tiup/blob/99e0aaf4d3f2e6598cce6011909022b7e22dd024/pkg/cluster/manager/basic.go#L310